### ISSUE TITLE
Can't open field slide-out from block config

### ISSUE THREAD

**Issue by adrienpicard** [verbb/vizy#321]
### Describe the bug

Trying to edit another field from the Vizy config returns an error. The slide-out opens blank with an `A server error occurred.` message.

Here's what it says in the console.

```
file:  "/path/vendor/craftcms/cms/src/controllers/FieldsController.php"
line: 246
message: "Call to undefined method verbb\\vizy\\models\\BlockType::fieldLayouts()"
```

### Steps to reproduce

1. Open a Vizy field config
2. From there, open a Vizy Block config
3. Double click any field in the block (this works, slide out opens as expected)
4. Double click "original" field at the top to trigger the error

### Craft CMS version

5.4.9

### Plugin version

3.0.4

### Multi-site?

No

**Comment by engram-design** [comment]
Fixed for the next release. To get this early, run `composer require verbb/vizy:"dev-craft-5 as 3.0.4"`.

**Comment by engram-design** [comment]
Fixed in [3.0.5](https://github.com/verbb/vizy/releases/tag/3.0.5)
